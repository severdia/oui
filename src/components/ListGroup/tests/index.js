import React from 'react';
import {
  ListGroup,
  ListGroupItem,
} from '../index';
import { shallow } from 'enzyme';
import { shallowToJson } from 'enzyme-to-json';

describe('components/ListGroup', () => {

  it('render listgroup with correct classes', () => {
    const component = shallow(
      <ListGroup
        title="Hello Title"
        subtitle="Hello Title">
        <ListGroupItem>
          Item!
        </ListGroupItem>
        <ListGroupItem>
          Item 2!
        </ListGroupItem>
      </ListGroup>
    );
    expect(component.is('div')).toBe(true);
    expect(shallowToJson(component)).toMatchSnapshot();
  });

});
