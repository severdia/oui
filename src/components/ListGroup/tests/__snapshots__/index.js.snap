// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components/ListGroup render listgroup with correct classes 1`] = `
<div
  className="oui-listgroup flex soft--top soft--bottom border--bottom"
  data-test-section=""
>
  <div
    className="width--1-4"
  >
    <div
      className="zeta muted weight--normal"
    />
    <div
      className="epsilon line--tight force-break"
    >
      Hello Title
    </div>
    <div
      className="micro muted push-half--ends"
    >
      Hello Title
    </div>
  </div>
  <div
    className="group flex--1 soft--left"
  >
    <ListGroupItem
      density="loose"
    >
      Item!
    </ListGroupItem>
    <ListGroupItem
      density="loose"
    >
      Item 2!
    </ListGroupItem>
  </div>
</div>
`;
